docs/exploration-and-organization/events-and-timelines.md

```diff
@@ -12,19 +12,21 @@ Events and timelines are a way to capture that chronological knowledge and make
 
 ## Events
 
+![An event on a time series](./images/event-on-a-time-series.png)
+
 An event is basically a date + a title + a description + an icon. You can add events to Metabase to show important milestones, launches, or anything else, right alongside your data.
 
 ## Timelines
 
-You can group events into timelines, and associate those timelines with [collections](collections.md). For example, you may want to have a timeline that contains important email or sales dates, or an outages timeline that tracks downtime. You can move events between timelines, and move timelines from collection to collection.
+Timelines are groups of events associated with a [collection](collections.md).
 
-Collections can have timelines, and timelines can contain events. In practice what this means is that events you've added to a timeline will show up on time series questions stored in the same collection as that timeline. Events are invisible on dashboards
+![Timeline sidebar](./images/timeline-sidebar.png)
 
-## Event and timeline permissions
+For example, you may want to have a timeline that contains important email or sales dates, or an outages timeline that tracks downtime. You can move events between timelines, and move timelines from collection to collection.
 
-Anyone with [view access](../permissions/collections.md) to a collection can view the collection's events and timelines.
+Collections can have timelines, and timelines can contain events. In practice what this means is that events you've added to a timeline will show up on time series questions stored in the same collection as that timeline.
 
-Anyone with curate access to a collection can add events and timelines to that collection.
+Events are invisible on dashboards.
```

Comment:
Maybe move this to the "Events" section

Reply:
Also maybe slightly rephrase: "events are invisible on charts on dashboards" or something like this? I remember it was confusing to me what this sentence meant

Reply:
Maybe even call out explicitly: You can only see events on individual questions, not dashboards